app/experimenter/experiments/tests/test_bugzilla.py

```diff
@@ -44,13 +50,23 @@ def test_creating_pref_bugzilla_ticket_returns_ticket_id(self):
                 "cc": settings.BUGZILLA_CC_LIST,
                 "type": "task",
                 "priority": "P3",
+                "see_also": [
+                    get_bugzilla_id(experiment.data_science_bugzilla_url)
+                ],
+                "blocks": [get_bugzilla_id(experiment.feature_bugzilla_url)],
+                "url": experiment.experiment_url,
+                "whiteboard": experiment.STATUS_REVIEW_LABEL,
+                cf_tracking: "?",
```

Comment:
Why is cf_tracking "?" here?

Reply:
I took the issue description literally ```Set TrackingRelease{release}=?``` seeing that cf_status_firefoxXX has permissible values of ---, ?, +,  and -

Reply:
Perfect!